### PR TITLE
Update CHANGELOG.md for release 0.6.1

### PR DESCRIPTION
**Pull request by ademarco**

### CODE REVIEW

**Comment by ademarco** [comment]
:heart:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,19 @@
 # Change Log
 
+## [0.6.1](https://github.com/openeuropa/oe_theme/tree/0.6.1) (2018-08-13)
+[Full Changelog](https://github.com/openeuropa/oe_theme/compare/0.6.0...0.6.1)
+
+**Merged pull requests:**
+
+- OPENEUROPA-772: Update compose.json to have openeuropa/drupal-core-require-dev. [\#118](https://github.com/openeuropa/oe_theme/pull/118) ([voidtek](https://github.com/voidtek))
```
Comment (pfrenssen) [inline CHANGELOG.md]: I would update this message to something that explains the reasoning why this change was made, that would be more informative to the persons reading the changelog rather than than some kind of dependency was added for unknown reasons.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
+- OPENEUROPA-772: Move styleguide into require-dev. [\#117](https://github.com/openeuropa/oe_theme/pull/117) ([richardcanoe](https://github.com/richardcanoe))
```
Comment (pfrenssen) [inline CHANGELOG.md]: I would here explain why the style guide has been moved. For example:

> OPENEUROPA-772: The Style Guide module has been moved to a development dependency since it is not required for using the theme in production environments.
